src/components/frame/useFrame.ts

```diff
@@ -9,11 +9,8 @@ type Status = "DISCONNECTED" | "CONNECTING" | "CONNECTED";
 
 interface UseParentFrameProps {
   dispatch: HostActionsHandler;
-  methods?: LegacyHostActions;
 }
-export const useParentFrame = function(
-  props: UseParentFrameProps
-): [boolean, Partial<LegacyFrameActions> & { dispatch: FrameActionsHandler }] {
+export const useParentFrame = function(props: UseParentFrameProps): [boolean, { dispatch: FrameActionsHandler }] {
```

Comment:
```suggestion
export const useParentFrame = function({dispatch}: UseParentFrameProps): [boolean, { dispatch: FrameActionsHandler }] {
```

Why not just destructure it?

Reply:
changed :)